### PR TITLE
remove respondd data which is not mesh relevant

### PR DESCRIPTION
**Pull request by genofire**
Those data should not be provided by mesh protocol packages.
(Most of them is duplicated)

DO NOT MERGE without Merge of  https://github.com/freifunk-gluon/packages/pull/188

### CODE REVIEW

**Comment by NeoRaider** [comment]
Also, when the providers are unified, the current data format generated by the batadv provider should be preserved for compatiblity.

**Comment by genofire** [comment]
okay at first only the wifi clients?

**Comment by genofire** [comment]
Thats what i am working on https://github.com/freifunk-gluon/packages/pull/188

**Comment by genofire** [comment]
@christf current state will remove `statistics.clients.wifi` at babel - batman has it own method to count them

**Comment by NeoRaider** [comment]
As mentioned in freifunk-gluon/packages#188, I think this information should be moved to the gluon-respondd package.

**Comment by genofire** [comment]
Just for cleanup, in first step only: https://github.com/freifunk-gluon/packages/pull/217

enhancement comes later in: https://github.com/freifunk-gluon/packages/pull/188

File: package/gluon-mesh-batman-adv/src/respondd.c [hunk 1]
```diff
@@ -444,74 +444,6 @@ static struct json_object * get_traffic(void) {
 	return ret;
 }
 
-static void count_iface_stations(size_t *wifi24, size_t *wifi5, const char *ifname) {
```
Comment (AiyionPrime) [inline package/gluon-mesh-batman-adv/src/respondd.c:447]: `count_iface_stations` has already been moved to respondd-neighbours.c in 0f1fa243f

File: package/gluon-mesh-batman-adv/src/respondd.c [hunk 2]
```diff
@@ -444,74 +444,6 @@ static struct json_object * get_traffic(void) {
...
-	struct iwinfo_assoclist_entry *entry;
-	for (entry = (struct iwinfo_assoclist_entry *)buf; (char*)(entry+1) <= buf + len; entry++) {
-		if (entry->inactive > MAX_INACTIVITY)
-			continue;
-
-		(*wifi)++;
-	}
-}
-
-static void count_stations(size_t *wifi24, size_t *wifi5) {
```
Comment (AiyionPrime) [inline package/gluon-mesh-batman-adv/src/respondd.c:478]: As well as `count_stations`

**Review by AiyionPrime — CHANGES_REQUESTED** [review]
I think everything wifi-related has been purged from [gluon-mesh-babel](https://github.com/freifunk-gluon/gluon/tree/master/package/gluon-mesh-babel) and [gluon-mesh-batman-adv](https://github.com/freifunk-gluon/gluon/blob/master/package/gluon-mesh-batman-adv/src/respondd.c)

@genofire Maybe you can refresh the PR to point out what I'm overlooking or close the PR to reflect it's done.

**Comment by genofire** [comment]
This PR is so much outdated and the subject is done.
So lets close it.

Maybe we make a new PR to remove it again, when we/i solve make that parts independent from gluon (without libgluonutils) by:

- https://github.com/freifunk-gluon/packages/pull/217
- https://github.com/freifunk-gluon/packages/pull/188

**Comment by AiyionPrime** [comment]
> This PR is so much outdated and the subject is done.
So lets close it.

What a great idea :P

But thanks for closing it, highly appreciated.

**Pull request closed without merging**